src/sentry/integrations/source_code_management/commit_context.py

```diff
@@ -352,6 +352,36 @@ def queue_comment_task_if_needed(
 
                     pr_comment_workflow.queue_task(pr=pr, project_id=group_owner.project_id)
 
+    def queue_open_pr_comment_task_if_needed(
+        self, pr: PullRequest, organization: Organization
+    ) -> None:
+        try:
+            open_pr_comment_workflow = self.get_open_pr_comment_workflow()
+        except NotImplementedError:
+            return
+
+        if not OrganizationOption.objects.get_value(
+            organization=organization,
+            key=open_pr_comment_workflow.organization_option_key,
```

Comment:
For GitHub, this is `sentry:github_open_pr_bot` as before. See https://github.com/jianyuan/sentry/blob/ref/queue_open_pr_comment_task_if_needed/src/sentry/integrations/github/integration.py#L452